includes/admin-menu.php

```diff
@@ -222,6 +222,26 @@ function bt_custom_toolbar_links( $wp_admin_bar ) {
 				),
 			);
 			$wp_admin_bar->add_node( $args );
+
+			if ( strpos( $_SERVER['REQUEST_URI'], '?' ) ) {
+				$query_char = '&';
+			} else {
+				$query_char = '?';
+			}
+
+			if ( isset( get_option( 'blog_tutor_support_settings' )['relay_key'] ) && isset( get_option( 'blog_tutor_support_settings' )['relay_url'] ) ) {
```

Comment:
Can we add this logic to a method in the NerdPress_Helpers Class? For better reusability.